tests/foreman/ui/test_contentview.py

```diff
@@ -3867,3 +3867,73 @@ def test_positive_inc_publish_cv(session, module_org):
         )
         cvv = entities.ContentView(id=cv.id).read().version[1].read()
         assert cvv.package_count == 32
+
+
+@pytest.mark.tier2
+def test_positive_able_to_disable_and_enable_rhel_repos(session, target_sat):
+    """Upstream repo name changes shouldn't negatively affect a user's ability
+    to enable or disable a repo
+
+    :id: 205a1c05-2ac8-4c60-8d09-016bbcfdf538
+
+    :expectedresults: User is able to enable and disable repos without issues.
+
+    :BZ: 1973329
+
+    :customerscenario: true
+
+    :CaseAutomation: Automated
+    """
+    org = entities.Organization().create()
+    with manifests.clone() as manifest:
+        upload_manifest(org.id, manifest.content)
+    # rhel7
+    rhel7_repo = target_sat.cli_factory.RHELRepository()
```

Comment:
Current implementation works for `rhel8_bos`, but it's not as clean as overriding `cli_factory.RHELRepository()` to use `rhel8_bos`, which I have no idea how to do.  I've tried passing in "`disto='rhel8', resositories=target_sat.cli_factory.make_repository()`" without much success.